metadata-proxy/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM gcr.io/google_containers/nginx
+MAINTAINER Quintin Lee <qlee@google.com>
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+  iptables \
+  sudo
```

Comment:
I think it's weird to run sudo in a container... it should probably just run as root. If you're trying to avoid running nginx as root, I would just switch to a different user for that command.